hw3/hw3.playground/Contents.swift

```diff
@@ -0,0 +1,464 @@
+import UIKit
+
+public enum Location {
+    case RUSSIA
+    case JAPAN
+    case Italy
+}
+
+
+public enum Manufacture {
+    case Lada
+    case Lexus
+    case Ferrari
+}
+
+
+public enum TypeCar {
```

Comment:
По аналогии с CarState

```suggestion
public enum CarType {
```